importService/serverless.yml

```diff
@@ -60,4 +82,48 @@ functions:
           rules:
             - prefix: ${env:IMPORT_FILE_PREFIX}
             - suffix: .csv
-          existing: true 
+          existing: true
+
+  catalogBatchProcess:
+    handler: handler.catalogBatchProcess
+    events:
+      - sqs:
+          batchSize: 5
+          arn:
+            Fn::GetAtt:
+              - catalogItemsQueue
+              - Arn
+
+resources:
+  Resources:
+    catalogItemsQueue:
+      Type: AWS::SQS::Queue
+      Properties:
+        QueueName: catalogItemsQueue
+    createProductTopic:
+      Type: AWS::SNS::Topic
```

Comment:
SNS and SQS topics should be done in product service. If in future we will add new import service, in that approach we will need to build the same logic in new service. But if sns and sqs will exist in product service - all other services will be able to use it.

Reply:
get your point. I'll move them in scope of next task(or may be today in the evening when get home). For now I suppose it is not a big deal to move this code and configuration into another package